config/envoy.yaml

```diff
@@ -55,5 +55,5 @@ static_resources:
             - endpoint:
                 address:
                   socket_address:
-                    address: localhost
+                    address: backend-v2
```

Comment:
plan here is:
merge this PR (=> keeping the history) and then create a duplicate of the previous state in the deployment folder